daemon/daemon_unix.go

```diff
@@ -1184,3 +1194,30 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) initCgroupsPath(path string) error {
+	if path == "/" || path == "." {
+		return nil
+	}
+
+	daemon.initCgroupsPath(filepath.Dir(path))
+
+	root := "/sys/fs/cgroup/cpu"
+	path = filepath.Join(root, path)
+	sysinfo := sysinfo.New(false)
+	if err := os.MkdirAll(path, 0755); err != nil && !os.IsExist(err) {
+		return err
+	}
+	if sysinfo.CPURealTimePeriod && daemon.configStore.CPURealTimePeriod != 0 {
```

Comment:
@erikstmartin just reread the code, indeed it just seems to be for the parent path.

In that case it makes sense, although I wish we didn't have to resort to do so :)